docs/api.rst

```diff
@@ -2,5 +2,5 @@ API documentation
 -----------------
 
 .. toctree::
-    modules/fullTN.rst
+    modules/general_state.rst
```

Comment:
I no longer think "state" should appear in the name of the module. My current preference is to return it to the top of the module and call it "Generic Tensor Network" in the docs.